github-activity-tracker/frontend/.env.example

```
@@ -0,0 +1,7 @@
+VITE_API_BASE_URL=http://localhost:3000
+RDS_HOST=your-rds-endpoint
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The RDS_HOST key should go before the VITE_API_BASE_URL key

(UnorderedKey)